### PR TITLE
Fix missing Operators.h include

### PR DESCRIPTION
**Pull request by peterbell10**
Currently `Operators.h` is included indirectly via `Tensor.h` but I'm removing this in pytorch/pytorch#68247

### CODE REVIEW

**Comment by peterbell10** [comment]
I tried a `torch_pin` but it doesn't seem to pick up docker file changes needed to build the corresponding PR. This PR works fine without the pin though.

**Comment by JackCaoG** [comment]
@peterbell10 We could just merge this pr after all test passed and you can rerun the xla test on your pytorch pr 2~3 hours after this pr is merged(so head rebuild a docker image for pytorch pr to pull)

**Review by JackCaoG — APPROVED** [review]

**Pull request merged**
